### PR TITLE
Links OOC between the two servers.

### PR DESCRIPTION
**Pull request by Iamgoofball**
:cl: Iamgoofball
experiment: OOC is now linked between the two servers.
/:cl:

if I say farts in OOC on Sybil, Basil will also hear farts in OOC.

If you want to mute someone just click the server hop button and mute them on that server until I code the cross-server mute button

### CODE REVIEW

**Comment by Isratosh** [comment]
NO

**Comment by francinum** [comment]
GOOF

**Comment by Sligneris** [comment]
Ahhahahahhaahaha!

No.

**Comment by Jalleo** [comment]
Not until its split to global OOC and Server OOC and we have a mute for it.

**Comment by Cheridan** [comment]
http://i.imgur.com/oqmg5MD.png

**Comment by Jalleo** [comment]
Its true I ask it and then I get this hilarious one.

**Comment by bgobandit** [comment]
ick ock becomes horribly confused

**Comment by WJohn** [comment]
:+1: x :100: 
Actually pretty funny. More confused OOC spam is always good.

File: code/world.dm [hunk 1]
```diff
@@ -161,7 +161,15 @@ var/last_irc_status = 0
 				minor_announce(input["message"], "Incoming message from [input["message_sender"]]")
 				for(var/obj/machinery/computer/communications/CM in machines)
 					CM.overrideCooldown()
-
+#define CHAT_OOC 1
+			if(input["crossmessage"] == "OOC")
+				for(var/client/C in clients)
```
Comment (xxalpha) [inline code/world.dm]: there's only clients in clients :clapper:

Comment (Iamgoofball) [inline code/world.dm]: according to all laws of aviation

**Comment by PKPenguin321** [comment]
this is actually really cool but at the same time it could be bad for stuff like OOC getting flooded at round end and stuff
maybe make a GOOC verb for global OOC that is cross server?

**Comment by Iamgoofball** [comment]
goofOOC

**Comment by PKPenguin321** [comment]
no but seriously you should do that
this could be really neat but shouldnt be mashed into regular OOC
make sure it has a toggle too

**Comment by Jalleo** [comment]
Already been mentioned by MrP and Aran. PK

**Comment by kevinz000** [comment]
GO HOME YOU'RE DRUNK BAGIL DOES NOT NEED TO HEAR OUR SYBILLIAN SHITPOSTING AND NEITEHR DO WE NEED TO HEAR THEIR ARPI.

**Comment by kevinz000** [comment]
Yes. Cross server shitpost lets go :D

**Comment by LaharlMontogmmery** [comment]
Right, I'm going to pipe in for this one as a admin : 

- First of all, have a way to differentiate Sybil OOC and Basil OOC. Like adding [S] or [B] before each name.
- Split up Global OOC and local OOC - as in, add a GOOC verb that will send to the other server.
- Finally, have an option to toggle listening to GOOC, like with Local OOC.

Also, a suggestion : if you can implement such a system, can you implement a GASAY? It'd be great if admins online could communicate between servers for help if needed.

**Comment by xxalpha** [comment]
Do all those things @LaharlMontogmmery suggested, before merge preferably.

**Comment by optimumtact** [comment]
Pretending this will get merged is the real meme.

File: code/world.dm (hunk 1 above)
Comment (ChangelingRain) [inline code/world.dm]: >according to all laws of aviation

goof should not be able to make prs

and yet

**Pull request closed without merging**